### PR TITLE
add json for team

### LINKED ISSUE #2: Setup data structure to store info about team members
Tasks:

- [ ] Team member details would be stored in JSON file as an object inside an array
- [ ] Create a sample JSON file that contains details of at least two members (dummy data), showing the fields which should be added in the team listing page.

### PR DESCRIPTION
**Pull request by vivekshah1801**
Resolves #2  

Added JSON file for team details.
Have put the file in 'data' folder for organized project structure.

### CODE REVIEW

File: data/team.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "name":"John Doe",
```
Comment (YashKumarVerma) [inline data/team.json:3]: add a role property also. admin, maintainer, contributor etc

Comment (YashKumarVerma) [reply data/team.json:3]: also consider leaving new line at end of file

**Comment by vivekshah1801** [comment]
@YashKumarVerma please check. I updated the code.

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions.

**Pull request merged**